packages/modal-content/src/ModalContentState.tsx

```diff
@@ -0,0 +1,41 @@
+import { ForwardedRef, forwardRef } from 'react'
+import { ModalContentStateProps, ModalContentStateEnum } from './types'
+
+import {
+  ModalContentStateStyle,
+  ModalContentStateIconStyle,
+} from './ModalContentStateStyles'
+
+import { Loader } from '@lidofinance/loaders'
+import { Check, Close } from '@lidofinance/icons'
+
+function ModalContentState(
+  props: ModalContentStateProps,
+  ref?: ForwardedRef<HTMLDivElement>
+) {
+  const { state, children, ...rest } = props
+
+  let stateContent
```

Comment:
As per our best practices, these values should be wrapped into a `useMemo` hook to avoid unnecessary calculations, e.g.
```js
const stateContent = useMemo(() => {
    switch(state) {
        case ModalContentStateEnum[ModalContentStateEnum.loading]:
            return stateContent = <Loader size={'large'} />
        // ...
    }
}, [state])
```

Reply:
Thanks, i will use for optimization :)

so, it is not expensive computation, why optimization via useMemo here?

Reply:
It's not expensive but it's a good habit to pick up